R/dataDrivenClusters.R

```diff
@@ -5,34 +5,44 @@
 dataDrivenClusters <- function(voxel_df, n_pca = 20, n_umap = 2, n_clust = 2,
```

Comment:
When I try to run the function, I get an error because the irlba and umap functions could not be found. Is there any way to import them when installing? If I'm not mistaken, I should not have to call `library(irlba)` or `library(umap)` to run the function.

Reply:
I believe I have to run devtools::document() and the namespace will update, so those packages will be included. However when I do this I run up with the following error:
`Warning messages:
1: -- Conflicts ----------------------------------------------------------------------- VoxelProc conflicts
--
x `dataDrivenClusters` masks `VoxelProc::dataDrivenClusters()`.
i Did you accidentally source a file rather than using `load_all()`?
  Run `rm(list = c("dataDrivenClusters"))` to remove the conflicts. 
2: Skipping NAMESPACE
x It already exists and was not generated by roxygen2. 
3: Skipping NAMESPACE
x It already exists and was not generated by roxygen2.`

I'm not sure what's going on here, but it appears as if there is somehow a second function called dataDrivenClusters that is masking the function that we are working with.

Reply:
Okay I've fixed the documentation and the namespace is updated.